src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -1782,4 +1782,33 @@ Please specify one of them using the --source option to proceed.</value>
   <data name="FileNotFound" xml:space="preserve">
     <value>File not found.</value>
   </data>
+  <data name="WindowsFeatureNotFound" xml:space="preserve">
+    <value>The feature [{0}] was not found.</value>
+    <comment>{Locked="{0}"} Error message displayed when a Windows feature was not found on the system.</comment>
+  </data>
+  <data name="FailedToEnableWindowsFeatureOverrideRequired" xml:space="preserve">
+    <value>Failed to enable Windows Feature dependencies. To proceed with installation, use --force</value>
+    <comment>{Locked="--force"}</comment>
+  </data>
+  <data name="RebootRequiredToEnableWindowsFeatureOverrideRequired" xml:space="preserve">
+    <value>Reboot required to fully enable the Windows Feature(s); to override this check use --force</value>
+    <comment>"Windows Feature(s)" is the name of the options Windows features setting.</comment>
+  </data>
+  <data name="FailedToEnableWindowsFeatureOverridden" xml:space="preserve">
+    <value>Failed to enable Windows Feature dependencies; proceeding due to --force</value>

```

Comment:
I left this as is because there are many other examples where `proceeding due to --arg` is used such as:
InstallerHashMismatchOverridden
PortableHashMismatchOverridden
ArchiveFailedMalwareScanOverridden

I think this is fine because it tells the user what arg they included to override the check. I'm also not sure what else to put so if you feel strongly about this, I can change this (and the others as well) to follow a different pattern.